types.cc

```diff
@@ -2011,6 +2011,13 @@ struct is_byte_order_equal_visitor {
 
 bool abstract_type::is_byte_order_equal() const { return visit(*this, is_byte_order_equal_visitor{}); }
 
+bool abstract_type::equal(bytes_view v1, bytes_view v2) const {
+    if (is_byte_order_equal()) {
+        return compare_unsigned(v1, v2) == 0;
```

Comment:
This adds a virtual call, since in final types this would be inlined, and here it cannot be.

Reply:
fixed in the new version.